requirements-extra.txt

```diff
@@ -0,0 +1,5 @@
+neptune-client>=0.4.4
+comet-ml>=1.0.56
+mlflow>=1.0.0
+test_tube>=0.7.5
+wandb>=0.8.21
```

Comment:
add line in .github/workflow/ci-testing.yml
```
python -c "req = open('requirements-extra.txt').read().replace('>', '=') ; open('requirements-extra.txt', 'w').write(req)"
```
just belo the changing `requirements.txt` so we have the min version propoerly tested